internal/service/vpc/resource_aws_vpc_peering_connection.go

```diff
@@ -221,8 +221,8 @@ func resourceAWSVPCPeeringConnectionDelete(ctx context.Context, d *schema.Resour
 }
 
 func resourceAWSVPCPeeringConnectionImport(ctx context.Context, d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
-	if len(strings.Split(d.Id(), "/")) != 3 {
-		return nil, fmt.Errorf("invalid identifier %v, expected <project_name>/<vpc_id>/<aws_account_id>", d.Id())
+	if len(strings.Split(d.Id(), "/")) < 4 {
+		return nil, fmt.Errorf("invalid identifier %v, expected <project_name>/<vpc_id>/<aws_account_id>/<aws_vpc_id>[/<aws_vpc_region>]", d.Id())
```

Comment:
Should we have different format of error message for each `import` format?

Reply:
I used [ch. 12 of IEEE Std 1003.1](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html) to specify an optional parameter in the error message.

Reply:
would this always fail for this import format? As we would have the `len(strings.Split(d.Id(), "/")) == 3`
`terraform import aiven_aws_vpc_peering_connection.foo project_name/vpc_id/aws_account_id/aws_vpc_id`